src/js/edu-benefits/1990n/config/form.js

```diff
@@ -0,0 +1,36 @@
+// import _ from 'lodash/fp';
+
+// import fullSchema1990n from 'vets-json-schema/dist/ncs-benefits-schema.json';
```

Comment:
I really hate these names...wish it were something like `vets-json-schema/dist/form-22-1990n-schema.json`